pkgs/development/python-modules/tempest/default.nix

```diff
@@ -0,0 +1,77 @@
+{ lib, buildPythonApplication, fetchPypi
+, pbr, cliff, jsonschema, testtools, paramiko, netaddr, oslo-concurrency, oslo-config, oslo-log, stestr, oslo-serialization, oslo-utils, fixtures, pyyaml, subunit, stevedore, prettytable, urllib3, debtcollector, unittest2
+, flake8-import-order, hacking, coverage, oslotest, pycodestyle
+}:
+
+buildPythonApplication rec {
+  pname = "tempest";
+  version = "28.0.0";
+
+  src = fetchPypi {
+    pname = "tempest";
+    inherit version;
+    sha256 = "24fcc0baa2044454b17b6b4aa2b1b19682cf95eb92ca38a2f289d3cbc488b170";
+  };
+
+  nativeBuildInputs = [
+    pbr
+  ];
+
+  propagatedBuildInputs = [
+    cliff
+    jsonschema
+    testtools
+    paramiko
+    netaddr
+    oslo-concurrency
+    oslo-config
+    oslo-log
+    stestr
+    oslo-serialization
+    oslo-utils
+    fixtures
+    pyyaml
+    subunit
+    stevedore
+    prettytable
+    urllib3
+    debtcollector
+    unittest2
+  ];
+
+  checkInputs = [
+    stestr
+    flake8-import-order
+    hacking
+    coverage
+    oslotest
+    pycodestyle
+  ];
+
+  checkPhase = ''
+    runHook preCheck
+
+    mkdir -p bin
+    export PATH="$PWD/bin:$PATH"
```

Comment:
Are those not created under $out? Is upstream nursing those entries in their config files?

Reply:
This is used only during tests not install. Tests expect this application to be available as such. Arch did the same here.

Reply:
Can you add a comment with that information? That would be great!

Reply:
Done.